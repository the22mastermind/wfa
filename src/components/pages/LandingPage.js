import React from 'react';
import widget from '../../assets/img/widget.svg';

const LandingPage = () => {
  return (
    <div className="landing-page-container">
      <div className="landing-page-content">
        <div className="landing-page-text">
          <div className="landing-page-title">Get real-time weather conditions and forecasts in your city</div>
          <a href="/signup" className="cta-btn">Get started</a>
        </div>
        <div className="landing-page-widget">
          <img src={widget} className="st-widget" alt="Weather widget"/>
        </div>
      </div>
    </div>
  );
}

export default LandingPage;
