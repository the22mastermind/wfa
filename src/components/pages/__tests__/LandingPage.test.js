import React from 'react';
import renderer from 'react-test-renderer';

import LandingPage from '../LandingPage';


it('matches the Landing Page snapshot', () => {
  const tree = renderer.create(<LandingPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
