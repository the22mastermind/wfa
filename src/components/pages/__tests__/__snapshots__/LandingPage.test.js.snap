// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the Landing Page snapshot 1`] = `
<div
  className="landing-page-container"
>
  <div
    className="landing-page-content"
  >
    <div
      className="landing-page-text"
    >
      <div
        className="landing-page-title"
      >
        Get real-time weather conditions and forecasts in your city
      </div>
      <a
        className="cta-btn"
        href="/signup"
      >
        Get started
      </a>
    </div>
    <div
      className="landing-page-widget"
    >
      <img
        alt="Weather widget"
        className="st-widget"
        src="widget.svg"
      />
    </div>
  </div>
</div>
`;
